.github/workflows/sdk-nft-packs.yml

```diff
@@ -0,0 +1,38 @@
+name: SDK NFT Packs 
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    # Set job outputs to values from filter step
+    outputs:
+      core: ${{ steps.filter.outputs.core }}
+      package: ${{ steps.filter.outputs.package }}
+    steps:
+    - uses: actions/checkout@v2
+    # For pull requests it's not necessary to checkout the code
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
```

Comment:
If you want to reference `core` as an output, I think you need to add this to the filters

```
core:
  - 'core/**'
```

Existing example here: https://github.com/metaplex-foundation/metaplex-program-library/blob/master/.github/workflows/sdk-token-vault.yml#L10-L29

Reply:
I purposely removed that reference since this SDK does not depend on core .. actually most don't anymore.

Reply:
Makes sense. Then, I think you need to remove other places referencing `core`

* Line 13: `core: ${{ steps.filter.outputs.core }}`
* Line 28: `needs.changes.outputs.core == 'true'`

Reply:
Yes good catch .. was an oversight .. fixed.